build/pmm-server-docker/playbook-install.yml

```diff
@@ -33,6 +27,16 @@
         enabled: no
         gpgkey: file:///etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY
 
+    - name: Postgres                   | Add PostgreSQL YUM repository
+      when: ansible_os_family == 'RedHat'
+      yum_repository:
+        name: percona-ppg-11-experimental
+        description: PostgreSQL YUM repository - x86_64
+        baseurl: http://repo.percona.com/ppg-11/yum/experimental/7/RPMS/x86_64
```

Comment:
it will work only till GA. After GA we need to enable repo for PPG via percona-release tool. Please leave comment about it because we can forget to change it. Also you need to change playbooks for AMI/OVF images.

Reply:
Done.